pkgs/tools/system/pcstat/default.nix

```diff
@@ -1,19 +1,17 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage {
+buildGoModule rec {
   pname = "pcstat-unstable";
```

Comment:
Do we keep the `unstable` part now that there is an actual versioning?

(granted it's an early version but [according to nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming) the `-unstable` suffix is used when fetching a git commit and not a version tag)

Reply:
You are right again. Thanks for the review.